Apromore-Core-Components/Apromore-Portal/src/main/java/org/apromore/portal/dialogController/MainController.java

```diff
@@ -128,14 +146,104 @@
     public PortalSession portalSession;
     private Map<String, PortalPlugin> portalPluginMap;
 
+    static {
+        String encPassphStr = ApromoreEnvUtils.getEnvPropValue(
+                SYMMETRIC_KEY_SECRET_ENV_KEY, "encryption passphrase could *not* be attained from environment");
+
+        MainController.encKey = encPassphStr;
+        LOGGER.info("\n\nInitialised main controller and obtained encKey from environment\n");
+
+        // For safety/clear
+        encPassphStr = null;
+    }
```

Comment:
Add some logic for the Keycloak feature flag.
Do we really want to fail classloading if the configuration isn't right?

Reply:
Refactored to security helper class - removed from static class initialiser block [DONE].